### ISSUE TITLE
[Universal banner]: QA: Landmark region should be defined for Universal banner

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9869]
### Description

Page Element:
Landmark Region for Universal banner

Issue Description:
If it is supposed to be used as banner region, this should be given within A banner role or HTML header tag that is equivalent for banner role.

User Impact:
When the page content is not covered within a labelled landmark region, screen reader users are unable to skim through the page and understand its structure.

Test Tool:
JAWS FF
Voiceover safari iPhone

### Component(s) impacted

Universal banner
(web component canary and web component staging)

### Browser

Safari, Firefox

### Carbon for IBM.com version

v1.40.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component Canary https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component staging https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
